### ISSUE TITLE
Support for Managing GuardDuty Organization Admin Accounts

### ISSUE THREAD

**Issue by bflad** [terraform-providers/terraform-provider-aws#13005]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

AWS recently announced Organizations support in GuardDuty. One part of that functionality is the ability to delegate administration to other accounts.

### New Resource

* aws_guardduty_organization_admin_account

### Potential Terraform Configuration

```hcl
resource "aws_guardduty_organization_admin_account" "example" {
  admin_account_id = 123456789012
}
```

### References

- [Announcement](https://aws.amazon.com/about-aws/whats-new/2020/04/amazon-guardduty-simplifies-multi-account-threat-detection-with-support-for-aws-organizations/)
- [User Guide](https://docs.aws.amazon.com/guardduty/latest/ug/guardduty_organizations.html)
- [`DisableOrganizationAdminAccount`](https://docs.aws.amazon.com/guardduty/latest/APIReference/API_DisableOrganizationAdminAccount.html)
- [`EnableOrganizationAdminAccount`](https://docs.aws.amazon.com/guardduty/latest/APIReference/API_EnableOrganizationAdminAccount.html)
- [`ListOrganizationAdminAccounts`](https://docs.aws.amazon.com/guardduty/latest/APIReference/API_ListOrganizationAdminAccounts.html)

**Comment by bflad** [comment]
Support submitted: https://github.com/terraform-providers/terraform-provider-aws/pull/13034

**Comment by bflad** [comment]
Support for this new resource has been merged and will release with version 2.60.0 of the Terraform AWS Provider, likely later today. 👍
